cmd/kubeadm/app/cmd/token.go

```diff
@@ -257,13 +260,24 @@ func RunCreateToken(out io.Writer, client clientset.Interface, cfgPath string, i
 	// otherwise, just print the token
 	if printJoinCommand {
```

Comment:
if --certificate-key is passed but not --print-join-command we should error out with:
"cannot use --certificate-key without --print-join-command"